create-liquid-app

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
I think what is script does is more align with `react-scripts` rather than `create-react-app`. As such, you might want to call this maybe `liquid-scripts` or more specifically `waves-scripts`  (given that it deals with bobtimus)? :)

Reply:
Hehe, thanks for the suggestion. The name was in jest, but that makes sense!